src/environments/environment.ts

```diff
@@ -9,6 +9,6 @@ export const environment = {
   couchAddress: window.location.protocol + '//' + window.location.hostname + ':2200/',
   centerAddress: 'earth.ole.org:2200',
   centerProtocol: 'https',
-  parentProtocol: 'https',
+  parentProtocol: 'planet-parent-protocol',
```

Comment:
@lmmrssa This is not overwritten by the docker script.  That only runs for the production version.  Please do not change as this will break our development environment.